app/assets/stylesheets/application.scss

```diff
@@ -17,19 +13,20 @@ a {
   overflow: auto;
 }
 
-.form-input {
-  margin-bottom: 30px;
+.banner h1 {
+  font-size: 2em;
 }
 
-.csv {
-  border-collapse: collapse;
-  table-border: 1;
-}
-
-.csv, .csv th, .csv td {
-  border: 1px solid black;
-  text-align: left;
+footer {
+  margin-top: 200px;
   padding: 10px;
+  bottom: 0;
+  font-size: 0.8em;
+  background-color: #eee;
+  text-align: center;
+  color: #bbb;
```

Comment:
Color literals like `#bbb` should only be used in variable declarations; they should be referred to via variable everywhere else.